### PR TITLE
Guvnor 3391 Enable settings.xml passed as argument

### PR DESCRIPTION
**Pull request by desmax74**
Enable settings.xml passed as argument,  fix for  the master

### CODE REVIEW

**Comment by desmax74** [comment]
Jenkins retest this

**Comment by desmax74** [comment]
@porcelli @mbiarnes

File: kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/compiler/external339/AFBaseConfigurationProcessor.java [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+import org.codehaus.plexus.component.annotations.Requirement;
+import org.slf4j.Logger;
+
+/**
+ * Used to open the API of the maven embedder
+ * Original version: https://maven.apache.org/ref/3.3.9/maven-embedder/xref/org/apache/maven/cli/configuration/ConfigurationProcessor.html
+ * IMPORTANT: Preserve the structure for an easy update when the maven version will be updated
+ */
+@Component(role = ConfigurationProcessor.class, hint = AFBaseConfigurationProcessor.HINT)
+public class AFBaseConfigurationProcessor implements AFConfigurationProcessor {
```
Comment (porcelli) [inline kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/compiler/external339/AFBaseConfigurationProcessor.java]: All looks good in the PR to me, the only thing that I'd change is the name of this class. The word "Base" sounds to me that this is an abstract type that should be extended... while this type is just a Simple or Default ConfigProcessor.

**Comment by desmax74** [comment]
@porcelli removed AFBase and changed File to Path in the AFSettingsXML

**Comment by desmax74** [comment]
jenkins retest this

**Comment by desmax74** [comment]
jenkins retest this

**Comment by porcelli** [comment]
jenkins retest this

**Pull request merged**